### DISCUSSION TITLE
Multi-monitor-like behavior on a single monitor/nested layouts

### DISCUSSION THREAD

**Discussion by nazar-pc** [qtile/qtile#3780] [Ideas]
I'm living on bleeding edge with a huge 65'' 7680x4320 TV as a monitor.

It is so big it is inconvenient to look at the top part all the time.

What I'd like to do is to slice monitor on Qtile level into two vertically stacked 7680x2160 "monitors" and being able to manage them somewhat independently, while still being able to move windows between the two.

I could slice monitor on Xorg level, but it would prevent me from watching videos in full screen, so I'd like to do it on window manager, such that floating windows and full screen videos can still cross boundaries of such "monitors".

Essentially I'm imagining something like a new layout that can have nested layouts, so I can configure it to have 2 50% rows and put other layouts into those. "Slice" layout already has another layout as a fallback (even though I didn't try it), so it doesn't seem to be something that was never done before.

Columns layout is what I'm using right now, but it doesn't allow having "empty" slots to force windows to not stretch to the very top in my case. It seems like having a fundamental split is easier to manage either way.

I acknowledge how unique my setup/challenge is, but as a happy user, contributor and supporter of Qtile since 2018 I'm happy to at least partially fund/put a bounty on such development.

This is probably similar to #2411, but I wanted to describe the full use case here.

Thoughts?

**Comment by elParaguayo** [comment]
Certainly feels like it's possible.

I'll take a look at `Slice` and see if it's possible to base something off that.

EDIT: Yeah, this is totally possible. I've got a really basic version up and running. It works by letting you define areas of the screen and placing individual layouts in each area. Maximising a window takes up the whole screen area.

**Reply by elParaguayo** [reply]
There's a very rough version in #3793 if anyone's interested.

**Reply by nazar-pc** [reply]
Looks interesting and... quite simple? Would it be possible to allow setting a list of layouts for each split or it is expected that multiple ScreenSplits can be defined with a unique combination of layouts in each?

**Reply by elParaguayo** [reply]
Let's continue this in the PR thread.
